megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -2302,64 +2314,45 @@ public static ToHitData toHit(IGame game, int attackerId, Targetable target) {
         }
 
         ToHitData losMods = los.losModifiers(game, eistatus, ae.isUnderwater());
-        // toHit = new ToHitData(ae.getCrew().getGunnery(), "gunnery skill");
-        ToHitData toHit = new ToHitData(0, "base");
+        // toHit = new ToHitData(ae.getCrew().getGunnery(), Messages.getString("WeaponAttackAction.")gunnery skill");
```

Comment:
Let's get rid of this commented out code.

Reply:
Done. 
I think I just managed to change this line using find and replace to set up my Messages.getString calls.  I plan to do much more cleanup on WeaponAttackAction in small doses.